src/js/queries/topicPageQuery.js

```diff
@@ -0,0 +1,23 @@
+const topicPageQuery = `
+  query topicPageQuery($pk:ID) {
```

Comment:
Small nit: we should call it `$id` instead of `$pk`. The latter is used in `servicePageQuery` to aid the transition from database ID to slug (and should probably just be removed now). In this case we're using graphql IDs, so I think it'll be less confusing.